InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/auto/AutoAlign.java

```diff
@@ -54,7 +52,7 @@ public void execute() {
     public boolean isFinished() {
```

Comment:
This command should only be finished if they let off the button. We shouldn't need to override isFinished

Reply:
If you implement my suggestions to the Buttons wired, you can remove this override to isFinished and make this command just override execute. While the button is held down it will continually call execute, then it will cancel when the button is released. then you can get rid of the weird command switching code in the DriveSubsystem.

Reply:
So you're saying we can get rid of the "DriveMethod" idea entirely and instead wire it up with commands?